core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -818,13 +818,12 @@ class DAGScheduler(
       }
     }
 
-    val properties = if (jobIdToActiveJob.contains(jobId)) {
-      jobIdToActiveJob(stage.jobId).properties
-    } else {
-      // this stage will be assigned to "default" pool
-      null
+    val activeJob = jobIdToActiveJob.get(stage.jobId).getOrElse(null)
```

Comment:
Just push a change to this same branch that overwrites with the new change. Then it can be tested.